### PR TITLE
Disable vertex cache checkbox with MTRSX

### PR DESCRIPTION
**Pull request by CookiePLMonster**
Meme PR tbh.

### CODE REVIEW

File: rpcs3/rpcs3qt/settings_dialog.cpp [hunk 1]
```diff
@@ -393,6 +393,10 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> guiSettings, std:
 
 	xemu_settings->EnhanceCheckBox(ui->multithreadedRSX, emu_settings::MultithreadedRSX);
 	SubscribeTooltip(ui->multithreadedRSX, tooltips.settings.multithreaded_rsx);
+	connect(ui->multithreadedRSX, &QCheckBox::clicked, [this](bool checked)
+	{
+		ui->disableVertexCache->setEnabled(!checked);
+	});
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/settings_dialog.cpp]:
```suggestion
	});
	ui->disableVertexCache->setEnabled(!ui->multithreadedRSX->isChecked());
```

Comment (CookiePLMonster) [reply rpcs3/rpcs3qt/settings_dialog.cpp]: Vertex cache is automatically disabled with MTRSX so this option does nothing.

Comment (Megamouse) [reply rpcs3/rpcs3qt/settings_dialog.cpp]: lmao think again

Comment (CookiePLMonster) [reply rpcs3/rpcs3qt/settings_dialog.cpp]: Yes, ideally then it should be forced on - but then again, you can crank up resolution scaling to 800%, enable strict rendering mode and then res scaling is "stuck" at 800%.

Comment (Megamouse) [reply rpcs3/rpcs3qt/settings_dialog.cpp]: ???
I meant the suggested change.

Comment (CookiePLMonster) [reply rpcs3/rpcs3qt/settings_dialog.cpp]: Your suggested change won't make it dynamically enabled/disabled as you switch the option though? Scroll down a few lines to see how `ui->scrictModeRendering` is set up.

Comment (Megamouse) [reply rpcs3/rpcs3qt/settings_dialog.cpp]: uhmm... you are supposed to add that in order to fix the initial state

Comment (CookiePLMonster) [reply rpcs3/rpcs3qt/settings_dialog.cpp]: ooh

**Pull request merged**